JS-Algorithms/challenges.js

```diff
@@ -118,9 +168,19 @@ Because the 12th index in the Fibonacci sequence is 144, and 144 has three digit
 ***** */
 
 const fibIndex = (n) => {
-  // YOUR CODE HERE...
+  let a = 1, b = 1, index = 2; // here i initialized the first two Fibonacci numbers and the index
```

Comment:
We should use more descriptive naming, this improves the readability for our team mates.